siliconcompiler/report/streamlit_viewer.py

```diff
@@ -575,12 +576,63 @@ def show_title_and_runs(title_col_width=0.7):
         streamlit.columns([title_col_width, 1 - title_col_width], gap="large")
 
     with title_col:
-        streamlit.title(f'{new_chip.design} dashboard', anchor=False)
+        streamlit.markdown(
+            '''
+            <head>
+                <style>
+                    .logo-container {
+                        display: flex;
+                        align-items: flex-start;
+                    }
+                    .logo-image {
+                        margin-right: 10px;
+                        margin-top: -10px;
+                    }
+                    .logo-text {
+                        display: flex;
+                        flex-direction: column;
+                        margin-top: -20px;
+                    }
+                    .text1 {
+                        color: #F1C437; /* Yellow color */
+                        font: IBM Plex Sans !important;
```

Comment:
how common is this font? We already bundle a font with SC: https://github.com/siliconcompiler/siliconcompiler/blob/main/siliconcompiler/data/RobotoMono/RobotoMono-Regular.ttf  would it be possible to embed it into the HTML and use it instead to avoid being dependent on another font?

Reply:
Just did it!